db/range_del_aggregator_test.cc

```diff
@@ -458,9 +468,11 @@ TEST_F(RangeDelAggregatorTest, AddGetTombstoneInterleaved) {
   RangeDelAggregator range_del_agg(bytewise_icmp, {} /* snapshots */,
                                    true /* collapsed */);
   AddTombstones(&range_del_agg, {{"b", "c", 10}});
-  auto tombstone = range_del_agg.GetTombstone("b", 5);
+  auto key = InternalKey("b", kMaxSequenceNumber, kTypeValue);
+  auto tombstone = range_del_agg.GetTombstone(key.Encode(), 5);
   AddTombstones(&range_del_agg, {{"a", "d", 20}});
-  Slice b = "b", c = "c";
+  ParsedInternalKey b = {"b", kMaxSequenceNumber, kMaxValue};
+  ParsedInternalKey c {"c", kMaxSequenceNumber, kMaxValue};
   VerifyPartialTombstonesEq(PartialRangeTombstone(&b, &c, 10), tombstone);
```

Comment:
I think these tests would benefit from cases where a tombstone has been split across multiple files. Perhaps that what you meant by more love.

Reply:
Yes, that's exactly what I was referring to. Didn't get around to doing this yet, but it is high on my TODO list.

Reply:
I've added a few more test cases that verify behavior in the presence of truncated tombstones. PTAL.

Reply:
Are you sure you pushed? I'm having trouble finding these new test cases.

Reply:
I pushed to the wrong branch. Sorry about that. Take another look.